modules/wolfgame.py

```diff
@@ -787,6 +795,189 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference."""
+    if not rest or (rest.isdigit() and int(rest) == 0):
+        if is_user_altpinged(nick)[0]:
```

Comment:
I'd probably save is_user_altpinged(nick) off to a variable -- or even better, decompose it into two variables, e.g. (is_altpinged, altping_amount) = is_user_altpinged(nick). Then you can use is_altpinged and altping_amount later in the function to give more semantic meaning to all of your random checks as opposed to indexing into some arbitrary tuple.